executers/dependenciesutils_test.go

```diff
@@ -2,13 +2,14 @@ package executers
 
 import (
 	"fmt"
-	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
-	"github.com/jfrog/jfrog-client-go/utils/log"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"reflect"
 	"testing"
+
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
```

Comment:
This change was done by my IDE(gofmt in fact) so let me know if you use a different tool for formatting the imports.